### PR TITLE
MVP linking of plain features to scenario

### PR DESCRIPTION
**Pull request by hotzevzl**
https://vizzuality.atlassian.net/browse/MARXAN-547

only to unblock testing of the platform for demo scenarios - lacks tests and stuff

### CODE REVIEW

File: api/apps/api/src/modules/geo-features/geo-feature-set.service.ts [hunk 1]
```diff
@@ -115,33 +116,43 @@ export class GeoFeatureSetService {
...
+            RemoteFeaturesData,
+          );
+          const featuresData = await featuresDataRepo
+            .find({ featureId: feature.featureId })
+            .then((result) =>
+              result.map((fd) => ({
+                scenarioId,
+                featuresDataId: fd.id,
+                fpf:
+                  feature.marxanSettings?.fpf ??
```
Comment (kgajowy) [inline api/apps/api/src/modules/geo-features/geo-feature-set.service.ts:146]: Seems like a good candidate to be a default on entity/column instead scattering this logic within code. As anyone can just grab repo and add rows there, the default within service code could be overlooked.

**Review by kgajowy — COMMENTED** [review]
Would be great to add missing integration test for the changes as well.

File: api/apps/api/src/modules/geo-features/geo-feature-set.service.ts [hunk 2]
```diff
@@ -115,33 +116,43 @@ export class GeoFeatureSetService {
...
-          return repository.save({
-            scenarioId,
-            featuresDataId: feature.featureId,
-            fpf:
-              feature.marxanSettings?.fpf ??
-              MarxanFeaturesMetadata.defaults.fpf,
-            prop: feature.marxanSettings?.prop,
-          });
+        .map(async (feature) => {
+          const featuresDataRepo = transactionalEntityManager.getRepository(
```
[the remark this answers is not in the document]
Comment (hotzevzl) [reply api/apps/api/src/modules/geo-features/geo-feature-set.service.ts]: you are right, removed in a23319548cc954cf86578993d959f3c5872d9ca3

File: api/apps/api/src/modules/geo-features/geo-feature-set.service.ts (hunk 1 above)
Comment (hotzevzl) [reply api/apps/api/src/modules/geo-features/geo-feature-set.service.ts:141]: it does, though at this stage I'm not sure we'll keep any defaults here (fpf *may* be undefined) - I'd revisit this later.

File: api/apps/api/src/modules/geo-features/geo-feature-set.service.ts [hunk 3]
```diff
@@ -115,33 +116,43 @@ export class GeoFeatureSetService {
...
   ): Promise<
-    ({
+    {
       scenarioId: string;
       featuresDataId: string;
-      fpf: number;
-      prop: number | undefined;
-    } & ScenarioFeaturesData)[]
+      fpf?: number | undefined;
+      prop?: number | undefined;
```
[the remark this answers is not in the document]
Comment (hotzevzl) [reply api/apps/api/src/modules/geo-features/geo-feature-set.service.ts]: it is, indeed

[the remark this answers is not in the document]
Comment (hotzevzl) [reply api/apps/api/src/modules/geo-features/geo-feature-set.service.ts]: 13c07726f8a4ca4d38460984991ced7d3a721f41

**Pull request merged**